Selenium.psd1

```diff
@@ -5,33 +5,97 @@
 #
 # Generated on: 8/31/2018
 #
+# Modified James O'Neill 30 Nov 2019
 
 @{
+ModuleVersion        = '2.3.1'
+GUID                 = 'a3bdb8c3-c150-48a8-b56d-cd4d43f46a26'
+Author               = 'Adam Driscoll'
+CompanyName          = 'Unknown'
+Copyright            = '(c) 2019 Adam Driscoll. All rights reserved.'
+Description          = 'Web automation using the Selenium Web Driver'
+ScriptsToProcess     = @('SeleniumClasses.ps1')
+RootModule           =   'Selenium.psm1'
+RequiredAssemblies   = @('./assemblies/WebDriver.dll', './assemblies/WebDriver.Support.dll')
+FunctionsToExport    = @(
+    'Start-SeChrome' ,
+    'Start-SeEdge' ,
+    'Start-SeNewEdge' ,
+    'Start-SeFirefox' ,
+    'Start-SeInternetExplorer' ,
+    'Start-SeNewEdge' ,
+    'Stop-SeDriver' ,
+    'Invoke-SeClick' ,
+    'Send-SeClick' ,
+    'Get-SeCookie' ,
+    'Remove-SeCookie' ,
+    'Set-SeCookie' ,
+    'Get-SeElement',
+    'Get-SeElementAttribute' ,
+    'Get-SeSelectionOption',
+    'Get-SeKeys' ,
+    'Send-SeKeys' ,
+    'Save-SeScreenshot' ,
+    'New-SeScreenshot'
+    'Invoke-SeScreenshot' ,
+    'Open-SeUrl' ,
+    'Get-SeWindow' ,
+    'Switch-SeWindow',
+    'Switch-SeFrame',
+    'Clear-SeAlert',
+    'SeOpen',
+    'SeType',
+    'SeShouldHave'  )
+AliasesToExport      = @(
+    'Chrome' ,
```

Comment:
@jhoneill I think we should change the alias to SeChrome/SeFirefox/SeIE etc 
this can conflict with Linux where the name of the executable is chrome/firefox etc and PowerShell is case insensitive 

-DM

Reply:
Yes, I agree. I put in short names early in the start of doing the DSL stuff, and later got to
`SeOpen  <<url>> -In Chrome` 
Really having aliases with the same name as the executables isn't a great idea, and I'll back those out.

Reply:
I've changed those aliases in one of yesterday's commits.